modules/troubleshooting-disabling-autoreboot-mco.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting/troubleshooting-operator-issues.adoc
+
+[id="troubleshooting-disabling-autoreboot-mco_{context}"]
+= Disabling Machine Config Operator from autorebooting
+
+When you use the Machine Config Operator to modify an NTP, registry, secret, SSH key, or other resource, {op-system-first} must be rebooted for the change to take effect. To avoid unwanted disruptions, you can modify the machine config pool to prevent autorebooting after the Operator makes changes to the machine config.
```

Comment:
MCO also reboots nodes when an automatic configuration change happens (for example, when `kube-apiserver-to-kubelet-signer` CA is rotated). 

This paragraph may mistakenly lead to think that reboots only happen for intentional changes and that pausing would only prevent reboots triggered by intentional changes, which is wrong.

I'd suggest to rephrase this so it also takes into account reboots triggered by automatic configuration changes, like `kube-apiserver-to-kubelet-signer` CA rotation.

Reply:
Great suggestion! I have reworded the intro to give more context that config changes might be manual or automatic.

Reply:
Thanks! LGTM